### DISCUSSION TITLE
Faster issue creation workflow (keyboard only)

### DISCUSSION THREAD

**Discussion by kennysong** [community/community#6340] [Projects and Issues]
The board view is fantastic and really useful when creating and managing lots of issues in a project. 

However, creating a new issue takes 3 careful clicks on relatively small targets, which causes some friction when I want to create several tasks in a row. 

I'd love to be able to create an issue fully from the keyboard, but there doesn't seem to be a shortcut for doing so. Please add one! 

A pop-up modal for viewing/editing an issue body would be great as well.

**Comment by MarkBennett** [comment]
Totally agree with you here!
